doc/Streaming.xml

```diff
@@ -590,7 +605,245 @@
+AIBAQMBFAcBAQgBAQoBAQsBCgAAACcAIQAe30C59UrClE0e27UP5h/Wty9UL8+dfzg+2ttmmo3kBAAAAC8A"
+]]></programlisting>
+                  </para>
+                  <para>
+                    The SSRC is a required component for SRTP encryption and shall be specified during the RTSP session setup.
+                    The SSRC shall be provided in the transport header of the RTSP SETUP response as described in RFC 2326 Section 12.39.
+                  </para>
+                </section>
+                <section>
+                  <title>Client Re-Keying with MIKEY</title>
+                  <para>
+                    The client may request the use of new keys for stream encryption.
+                    The SET_PARAMETER method shall be used with a MIKEY parameter to set the new encryption keys.
+                    The device shall not reset the ROC when changing keys.
+                    When rekeying, the RAND value and the SP (Security Policy) payload may be omitted by the client as defined in RFC 3830 section 4.5.
+                    <programlisting><![CDATA[SET_PARAMETER rtsp://<RTSP_URI> RTSP/1.0
+CSeq: 6
+Session: 15405670
+User-Agent: OmnicastRTSPClient/1.0
+Content-Type: text/parameters
+Content-Length: 145
+
+mikey: AQAFAGgCr8EBAADSvxgkAAAAAAoAAdOOK7UihqIBAAAAGAABAQEBEAIBAQMBFAcBAQgBAQoBAQsBCgA
+AACcAIQAepekjs88g+Q7AU6LAvRsoVyn18ZW1JuXI9qht4g6+BAAAAAIA
+]]>
+                    </programlisting>
+                  </para>
+                </section>
+                <section>
+                  <title>Multicast considerations</title>
+                  <para>
+                    When in multicast, multiple RTSP clients shall be able to retrieve changing keys as well as the SSRC and ROC of the stream.
+                    A new GET_PARAMETER request has been added to retrieve the MIKEY.
+                    This MIKEY message shall contain the <emphasis>current</emphasis> ROC, meaning the server cannot send a cached MIKEY message.
+                  </para>
+                  <para>
+                    Request:
+                    <programlisting><![CDATA[GET_PARAMETER rtsp://<RTSP_URI> RTSP/1.0
+CSeq: 4
+Session: 15405670
+User-Agent: OmnicastRTSPClient/1.0
+Content-Type: text/parameters
+Content-Length: 9
+
+mikey
+]]>
+                    </programlisting>
+
+                    Response:
+                    <programlisting><![CDATA[RTSP/1.0 200 OK
+CSeq: 4
+
+mikey: AQAFAGrVolgBAADdBcAoAAAAAAsA2/K83QArhBIKEGrVolg1GZvp7DPyFCdYmXABAAAAGwABAQEBEAIBAQM
+BFAQBDgcBAQgBAQoBAQsBCgAAACcAIQAe7OzS5umZMXHqaegZC3UkDwbC5NNpj4b8+fB6MROeBAAAAA0A
+]]>
+                    </programlisting>
+                  </para>
+                </section>
+                <section>
+                  <title>Error management</title>
+                  <para>
+                    If an error occurs during the key management, the device shall return a status code 463 Key management failure as defined in RFC 4567.
```

Comment:
There could be scenario where the SET_PARAMETER during re-keying can fail too and returns "451 Parameter Not Understood" error? Is this requirement covered somewhere else or better to write/include it here?